apps/backend/core/configuration/validation/utils/database.py

```
@@ -0,0 +1,362 @@
+
+        # Test basic operations
+        test_key = "__toolboxai_validation_test__"
+        await client.set(test_key, "test", ex=60)  # 60s expiry
+        test_value = await client.get(test_key)
+        await client.delete(test_key)
+
+        if test_value != "test":
+            return False, "SET/GET validation failed", 0.0, metadata
+
+        latency_ms = (time.time() - start_time) * 1000
+        return True, None, latency_ms, metadata
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)